### ISSUE TITLE
Prevent empty `experiment_apis` property in manifest

### ISSUE THREAD

**Issue by willdurand** [mozilla/addons-linter#4365]
This is the addons-linter side of https://bugzilla.mozilla.org/show_bug.cgi?id=1773301, we should emit an error when a `manifest.json` contains an empty `experiment_apis` property:

```
experiment_apis: {}
```

**Comment by ioanarusiczki** [comment]
@willdurand Is this on linter 5.8.0  ? -> using --privileged and mozillaAddons permission , I added "experiment_apis": {} in manifest.json but I get no errors.

**Comment by willdurand** [comment]
@ioanarusiczki no, it will be in 5.9.0 (not released yet)

**Comment by ioanarusiczki** [comment]
@willdurand Ok, then I'll re-check this until this Thursday if 5.9.0 lands.

**Comment by willdurand** [comment]
On its way: https://github.com/mozilla/addons-linter/releases/tag/5.9.0

**Comment by ioanarusiczki** [comment]
@willdurand With linter 5.9.0 the error is displayed. 

![error message is present](https://user-images.githubusercontent.com/33448286/174787393-e2817597-b99d-4073-9f6b-f121aa44a517.png)
